README.md

```diff
@@ -2,7 +2,7 @@
 
 # kong-plugin-url-rewrite
 
-This is a Kong middleware to log requests / responses using cerberus API.
+Kong API Gateway plugin for url-rewrite purposes.
```

Comment:
We should say what kind of problem we are trying to solve with this plugin and why we cannot solve with Kong's existing plugins.